### PR TITLE
Still schedule on preemptRoutine in case filterRoutine is not called by K8S

### PR DESCRIPTION
**Pull request by yqwang-ms**

### CODE REVIEW

File: example/request/design/request.yaml [hunk 1]
```diff
@@ -157,8 +157,14 @@ spec:
                   gpuNumber: 8
                 - podNumber: 1
                   gpuNumber: 4
-        # Setup gpu, cpu and memory isolation for container runtime.
         spec:
+          # PodPriority should be set proportional to above AffinityGroupPriority,
```
Comment (zhypku) [inline example/request/design/request.yaml]: Shall we mention the difference of the two priorities? e.g., one is for hived, one is for k8s.

File: pkg/scheduler/scheduler.go [hunk 2]
```diff
@@ -646,8 +649,41 @@ func (s *HivedScheduler) preemptRoutine(args ei.ExtenderPreemptionArgs) *ei.Exte
 		panic(internal.NewBadRequestError(fmt.Sprintf(
 			"Pod has already been binding to node %v",
 			podStatus.Pod.Spec.NodeName)))
-	} else if podStatus.PodState == internal.PodPreempting {
-		victims := podStatus.PodScheduleResult.PodPreemptInfo.VictimPods
```
Comment (zhypku) [inline pkg/scheduler/scheduler.go]: Do we still need to record the preemption result in filterRountine now?

File: example/request/design/request.yaml (hunk 1 above)
Comment (yqwang-ms) [reply example/request/design/request.yaml]: sure

---
In reply to: [398985275](https://github.com/microsoft/hivedscheduler/pull/11#discussion_r398985275) [](ancestors = 398985275)

File: pkg/scheduler/scheduler.go (hunk 2 above)
Comment (yqwang-ms) [reply pkg/scheduler/scheduler.go]: No need in fact, but just keep it there for consistent and future usage.

---
In reply to: [398993753](https://github.com/microsoft/hivedscheduler/pull/11#discussion_r398993753) [](ancestors = 398993753)

**Pull request merged**